src/components/ChatEntry.js

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+const ChatEntry = ({id, sender, body, timeStamp, liked, type, color, onLike}) => {
+  const entryClass = type === 'local' ? 'local' : 'remote';
+  const bubbleClass = type === 'local' ? 'local' : 'remote';
```

Comment:
If `type` is a string that contains either `"local"` or `"remote"`, could we remove these two variables and use `type` directly in their place?